backend/tracim_backend/tests/library/test_webdav.py

```diff
@@ -237,7 +235,10 @@ def test_unit__create_content__ok(
             b"Greek Salad\n",
         )
 
-        assert result, "Result should not be None instead {0}".format(result)
+        print("==============")
+        print(result)
+        print("--------------")
```

Comment:
aren't this missing debug lines?

Reply:
you right. Fixed with commit `837a55055668b4d136db8816c2c3eec55e1fb9d9`